api/nodes/serializers.py

```diff
@@ -935,6 +937,7 @@ def update(self, instance, validated_data):
                 auth=auth,
                 save=True,
             )
+            instance.owner.check_spam(user, {'addons_forward_node_settings__url'}, get_headers_from_request(request))
```

Comment:
Additionally, can you add this spam check to where the forward addon is being updated in the NodeSettingsUpdateSerializer as well? 

Now that we know of three places where this needs to be called in views/serializers, I'd look into moving this logic back to the model Forward NodeSettings.save or similar.